final_task/rss_reader/rss_exceptions.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3.8
+
+
+class Error(Exception):
+    """Base class for exceptions in this module"""
```

Comment:
Here and below: It is better to have `pass` inside if class does not contain any logic